### PR TITLE
added vnet module integration

### PR DESCRIPTION
**Pull request by satenderrathee**
# Pull Request Template

## 📲 What

Module integration for VNET, subnet, ngs, ngs rules, route table , route , flowlogs and storage account

## 🤔 Why

Stacks Ignite 

## 🛠 How

How have you implemented it?

## 👀 Evidence

Link to builds/artifacts/etc.

image

image

## 🕵️ How to test

Notes for QA

@IamAndyW

### CODE REVIEW

File: outputs_vnet.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+output "name" {
```
Comment (IamAndyW) [inline outputs_vnet.tf:1]: Can you prefix each output with the resource type? for example `vnet_`

File: outputs_vnet.tf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  description = "The resource name of the virtual network."
+  value       = module.avm_res_network_virtualnetwork.name
+}
+
+output "resource_guid" {
```
Comment (IamAndyW) [inline outputs_vnet.tf:6]: Do we need both of these? For peering you only need the resource id.

What is the difference between these two outputs?

File: variables_vnet.tf [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  default     = null
+  description = <<DESCRIPTION
+Specifies an AzureNetwork DDoS Protection Plan.
+
+- `id`: The ID of the DDoS Protection Plan. (Required)
+- `enable`: Enables or disables the DDoS Protection Plan on the Virtual Network. (Required)
+DESCRIPTION
+}
+
+# variable "tags" {
```
Comment (IamAndyW) [inline variables_vnet.tf:74]: Remove

File: outputs_vnet.tf (hunk 1 above)
Comment (satenderrathee) [reply outputs_vnet.tf]: updated

File: variables_vnet.tf (hunk 3 above)
Comment (satenderrathee) [reply variables_vnet.tf]: removed

**Review by IamAndyW — APPROVED** [review]

**Pull request merged**